config/jobs/kubernetes/sig-node/containerd.yaml

```diff
@@ -521,34 +491,6 @@ periodics:
   annotations:
     testgrid-dashboards: sig-node-containerd
     testgrid-tab-name: e2e-gci-multizone
-- interval: 2h
-  name: ci-cri-containerd-e2e-gce-stackdriver
```

Comment:
I don't think this is using an out of date branch?  It extracts `ci/latest`

Reply:
Makes sense.